profiles/connected_services.py

```diff
@@ -33,8 +34,63 @@ def _check_service_gdpr_query_configuration(service_connections):
                 f"Connected service: {service.name} does not have an API for querying data."
             )
 
+        if (
+            service.idp
+            and ServiceIdp.KEYCLOAK in service.idp
+            and not service.gdpr_audience
+        ):
```

Comment:
Technically yes. But I thought that it would be nice if the presence of the value would be validated if the admin checks the "Keycloak"-checkbox. Then when the Checkbox is removed from "Tunnistamo" we can be sure that the required information is already filled in the `gdpr_audience` field.

Reply:
That works.